spec/person_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../book'
+require_relative '../persons'
+
+describe Person do
+  context 'containing a list of names and ages' do
+    person = Person.new(45, 'Margaret')
+
+    it 'expects Margaret age to be 45' do
+      expect(person.age).to eq 45
+    end
+    it 'expects Person.name to be Margaret' do
+      expect(person.name).to eq 'Margaret'
+    end
+
+    it 'expect can_use_services to equal true' do
+      expect(person.can_use_services?).to eq true
+    end
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/217905924-70655453-ded2-4675-b161-e8f23302dc30.png)

- [ ] You have tested this method :clap: but you didn't cover all the scenarios :fearful: means what if

1. The person doesn't have an age more than or equal to 18
2. The person doesn't have parent_permission true
3. The person doesn't have an age more than or equal to 18 and doesn't have parent_permission true

Kindly cover all the scenarios to fully test your classes with all the exceptional or expected cases :+1: